src/frontend/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "baseUrl": ".",
+    "jsx": "preserve",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "es6",
+      "scripthost",
+      "es2015",
+      "es2016",
+      "es2017",
+      "es2018.intl",
+      "esnext.intl"
+    ],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "paths": {
```

Comment:
You can remove this property, none of if this are installed on this project :smiley:

Reply:
Thanks, you'll never guess from which project I copied this file :smile: